tests/stubtest_allowlists/py3_common.txt

```diff
@@ -217,6 +217,8 @@ webbrowser.UnixBrowser.remote_action  # always overridden in inheriting class
 webbrowser.UnixBrowser.remote_action_newtab  # always overridden in inheriting class
 webbrowser.UnixBrowser.remote_action_newwin  # always overridden in inheriting class
 wsgiref.types  # Doesn't exist, see comments in file
+xml.dom.xmlbuilder.ExpatBuilder
+xml.dom.xmlbuilder.ExpatBuilderNS
```

Comment:
Why are these whitelisted?

Reply:
I now found the discussion above. Can you add a comment that mentions https://github.com/python/mypy/pull/11411 to these lines?
```

Reply:
Yes!

> It seems that everyone somehow forgot about this :)

The PR rises from the dead!